webapp/src/components/surveyPost/textQuestion/textQuestion.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+
+import type {Question} from 'types/mattermost-webapp';
+
+import './style.scss';
+
+export type Props = {
+    question: Question;
+    responseChangeHandler: (questionID: string, response: string) => void;
+    disabled?: boolean;
+    value?: string;
+}
+
+function TextQuestion({question, responseChangeHandler, disabled, value}: Props) {
+    const changeHandler = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+        responseChangeHandler(question.id, e.target.value);
+    }, [question.id, responseChangeHandler]);
+
+    return (
+        <div className='TextQuestion vertical'>
+            <div className='questionTitle'>{question.text}</div>
```

Comment:
Could this be written using a label to improve accessibility?
```html
<label>
    {question.text}
    <input/>
</label>
```
That should make it so that, when a user navigates to the input with a screen reader, it should read out the question to them

Reply:
Done.